contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -15,16 +15,24 @@ FOUNDRY_PROJECT_SUFFIX="-compile"
 CONTRACTS_DIR="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; cd ../ && pwd -P )"
 export FOUNDRY_PROFILE="$PROJECT"$FOUNDRY_PROJECT_SUFFIX
 
+# first argument is the contract path, second argument is whether to use the --via-ir flag
 compileContract() {
   local contract
   contract=$(basename "$1")
   echo "Compiling" "$contract"
 
+  local viaIR=""
```

Comment:
Every contract already compiled with via-ir enabled. See the `ccip-compile` foundry profile

Reply:
got it. removing this logic